salesagility/SuiteCRM
# Fixed #8448 - Improve GridLayoutMetaDataParser::removeField()

## Description




Issue reference: #8448

Stores the return value of array_keys() in a variable and then passes
a reference to that variable as an argument to array_shift().

## Motivation and Context

The function GridLayoutMetaDataParser::removeField() causes PHP to throw a warning
because it is directly passing a function's return value by reference to another function.

## How To Test This

1. Create a package with a module in Module Builder
2. Add a new field to the module
3. Delete the new field
4. Check Apache error log for messages like
```
PHP Notice:  Only variables should be passed by reference in
/var/www/html/suitecrm/modules/ModuleBuilder/parsers/views/GridLayoutMetaDataParser.php on line 384
```

## Types of changes


### Final checklist
